### ISSUE TITLE
/clang:-1: linker command failed with exit code 1 (use -v to see invocation)

### ISSUE THREAD

**Issue by alexzheng** [kloder-games/godot-admob#44]
**OS target (Android/iOS):**
iOS 12

**Godot version:**
3.0.6 release

**Issue description:**

I build using 
scons p=iphone tools=no bits=64 target=release arch=arm64 -j 4  IPHONESDK="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS12.0.sdk/"

**Comment by Shin-NiL** [comment]
Are you able to compile the template without the Admob module?

**Comment by alexzheng** [comment]
Yes, without the admob module, the final lib can work without any problem.

**Comment by Shin-NiL** [comment]
Is there any other debug log information?
